core/domain/suggestion_registry.py

```diff
@@ -2097,6 +2098,8 @@ def to_frontend_dict(
                 self.rejected_translation_word_count),
             'first_contribution_date': (
                 self.first_contribution_date.strftime('%b %d, %Y')),
+            'first_contributed_in_days': int(
+                (datetime.date.today() - self.first_contribution_date).days),
             'last_contributed_in_days': int(
                 (datetime.date.today() - self.last_contribution_date).days)
```

Comment:
This calculation seems quite common within this file and its associated test file. Is there a way of moving this logic into its own function/method?

Reply:
done